src/styles/sidebar/components/filter-status.scss

```diff
@@ -0,0 +1,41 @@
+@use "../../mixins/layout";
+@use "../../mixins/molecules";
+@use "../../variables" as var;
+
+@mixin facet {
+  white-space: nowrap;
+}
+
+.filter-facet {
+  @include facet;
+  font-weight: bold;
+}
+
+.filter-facet--muted {
+  @include facet;
+  color: var.$color-text--light;
+  font-style: italic;
+}
+
+.filter-facet--pre {
+  @include facet;
+}
+
+.filter-status {
+  @include molecules.card;
+  margin-bottom: 1em;
+  padding: var.$layout-space--small;
+  justify-content: space-between;
+
+  &__text {
+    flex-grow: 1;
+    padding-right: var.$layout-space--xsmall;
+  }
+
+  &__header {
+    // Temporary
```

Comment:
If this is temporary, I presume there is a plan to change this. If so, what is the eventual plan?

Reply:
🤦 Whoops. Good catch.